app/editor/src/features/content/form/ContentStoryForm.tsx

```diff
@@ -39,6 +39,7 @@ export interface IContentStoryFormProps {
   content: IContentForm;
   contentType: ContentTypeName;
   savePressed?: boolean;
+  product: string;
```

Comment:
Change component property to isSummaryRequired: boolean. Default the value to true.

Reply:
Updated, thanks